src/Sonarr.Api.V3/ProviderResource.cs

```diff
@@ -38,8 +38,9 @@ public virtual TProviderResource ToResource(TProviderDefinition definition)
                 Message = definition.Message,
                 Tags = definition.Tags,
                 Fields = SchemaBuilder.ToSchema(definition.Settings),
-
-                InfoLink = string.Format("https://wiki.servarr.com/Sonarr_Supported_{0}",
+                
+                //sonarrFixed: Provider Wiki Links (supported-X => supported#X)/supported is an disambagation page. the # should be a header on the page with appropiate details/link
```

Comment:
Unnecessary comment?

Reply:
corrected up the comment
`// sonarr/supported is an disambiguation page. the # should be a header on the page with appropriate details/link`